services/server/server.go

```diff
@@ -713,6 +718,13 @@ func (s *Server) ChangeSettings(ctx context.Context, req *serverpb.ChangeSetting
 		s.checksService.CleanupAlerts()
 	}
 
+	if oldSettings.Telemetry.Disabled != newSettings.Telemetry.Disabled {
```

Comment:
could you please add comment here, it's not clear why do we try to collect templactes and checks even if we disable telemetry.

Reply:
Done